Resources/Textures/_Floof/LobbyScreens/attributions.yml

```
@@ -0,0 +1,14 @@
+- files: "hellshift.png"
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "JTCircus"
+  source: "JTCircus.com"
+
+- files: "nuke_club.png"
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "@Wab.Boi on Discord"
+  source: "https://boiwab.carrd.co"
+
+- files: "The_Maints_bar.png"
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "@layziilykaotic.bsky.social and @layziart"
+  source: "@layziilykaotic.bsky.social"
```

Seems like it is expecting a 'list' as the `files` and a URL as the `source`.

Try this.

```suggestion
- files: ["hellshift.png"]
  license: "CC-BY-NC-SA-3.0"
  copyright: "JTCircus"
  source: "https://JTCircus.com"

- files: ["nuke_club.png"]
  license: "CC-BY-NC-SA-3.0"
  copyright: "@Wab.Boi on Discord"
  source: "https://boiwab.carrd.co"

- files: ["The_Maints_bar.png"]
  license: "CC-BY-NC-SA-3.0"
  copyright: "@layziilykaotic.bsky.social and @layziart"
  source: "https://bsky.app/profile/layziilykaotic.bsky.social"
```

---

Trying it now, thanks!